pkgs/development/libraries/gpgme/default.nix

```diff
@@ -110,7 +105,16 @@ stdenv.mkDerivation rec {
     ++ lib.optional stdenv.hostPlatform.is32bit "-D_FILE_OFFSET_BITS=64"
   );
 
-  doCheck = true;
+  # prevent tests from being run during the buildPhase
+  makeFlags = [ "tests=" ];
+
+  # Tests will try to communicate with gpg-agent instance via a UNIX socket
+  # which has a path length limit. Nix on darwin is using a build directory
+  # that already has quite a long path and the resulting socket path doesn't
+  # fit in the limit. https://github.com/NixOS/nix/pull/1085
+  doCheck = !stdenv.isDarwin;
```

Comment:
`--disable-gpg-test` is not sufficient anymore?

Reply:
That does not seem to disable all of the tests; just some of them:

```
$ ./configure --help | grep test
  --cache-file=FILE   cache test results in FILE [disabled]
  --disable-glibtest      do not try to compile and run a test GLIB program
  --disable-gpgconf-test  disable GPGCONF regression test
  --disable-gpg-test      disable GPG regression test
  --disable-gpgsm-test    disable GPGSM regression test
  --disable-g13-test      disable G13 regression test
```

Reply:
Ah, but the other tests should remain enabled on Darwin.  I see.  Fixed in 1519ec729895.